packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -1107,7 +1113,13 @@ export class BackgroundScript {
     if (!this.buildConfig.isCI && !this.buildConfig.useLocalMockServer) {
       this.api.runtime.onInstalled.addListener(details => {
         if (details.reason === 'install') {
-          this.api.tabs.create({ url: `${this.coilDomain}/signup` })
+          void this.api.tabs.create({ url: `${this.coilDomain}/signup` })
+          let checkFor = EXTENSION_IDS.chrome
+          if (navigator.userAgent.includes('Firefox')) {
```

Comment:
Are you sure this works? Some other browsers actually include Firefox in their user agent string iirc ?

Reply:
Chrome: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.61 Safari/537.36